website/app/styles/pages/application/sidebar.scss

```diff
@@ -11,6 +11,11 @@ $doc-page-sidebar-transition-duration: 0.25s;
 
 // ⚠️ ---  the sidebar "navigation" is different between the homepage and the other pages --- ⚠️
 
+// if dialog is open, set sidebar z-index to 1
+.doc-page-wrapper:has(dialog[open]) {
+  --doc-z-index-sidebar: 1;
+}
+
```

Comment:
Let me know if you think this style should be moved somewhere else or if you think I should override the z-index value directly vs. overriding the token in this way...

Reply:
I think it's safe to move this on L66 or similar, inside `body.application:not(.index)` (as we're not going to need it for the index page),

Reply:
I moved it, thanks